tests/dashboard/test_get_chart_data.py

```diff
@@ -29,53 +29,230 @@ def mock_data_frame(filter_param):
 @mock.patch("src.dashboard.get_chart_data.get_chart_data._get_table_cols", mock_get_table_cols)
 @mock.patch.dict(os.environ, MOCK_ENV)
 @pytest.mark.parametrize(
-    "query_params,filters,path_params,query_str",
+    "query_params,filter_groups,path_params,query_str",
     [
         (
             {"column": "gender"},
             [],
             {"data_package_id": "test_study"},
-            f'SELECT gender, sum(cnt) as cnt FROM "{TEST_GLUE_DB}"."test_study" '
-            "WHERE COALESCE (cast(race AS VARCHAR)) IS NOT NULL AND gender IS NOT NULL "
-            "GROUP BY gender ORDER BY gender",
+            f"""SELECT
```

Comment:
these are getting particularly irritating to maintain - I did open #140 for considering ditching this in favor of an actual in-DB test.